sbnobj/Common/CRT/CRTPMTMatching.hh

```diff
@@ -33,6 +33,42 @@ namespace sbn::crt {
       exSide_enBottom   = 14, ///< Matched with one Bottom CRT hit before the optical flash and matched with one Side CRT hit after the optical flash.
       others            = 9  ///< All the other cases.
       };
+  MatchType assignFlashClassification(uint topen, uint topex, 
+				      uint sideen, uint sideex, 
+				      uint bottomen, uint bottomex)
```

Comment:
Some documentation of the function may be useful. Example:
```suggestion
  /**
   * @brief Returns the classification of a TPC object based on the CRT hits associated to it.
   * @param topen number of hits on the top CRT earlier than the object (entering)
   * @param topex number of hits on the top CRT later than the object (exiting)
   * @param sideen number of hits on any side CRT earlier than the object (entering)
   * @param sideex number of hits on any side CRT later than the object (exiting)
   * @param bottomen number of hits on the bottom CRT earlier than the object (entering)
   * @param bottomex number of hits on the bottom side CRT later than the object (exiting)
   * @return a value encoding the classification of the TPC object
   *   
   */
  MatchType assignFlashClassification(uint topen, uint topex, 
				      uint sideen, uint sideex, 
				      uint bottomen, uint bottomex)
```
You may also consider to add assumptions, like if having both entering and exiting  hits from the same side is expected/allowed and how it is considered (I suppose that's different from bottom, side and "top which is also a bit side"), if more than one hit are allowed per side, etc.

Reply:
i added some documentation in this commit https://github.com/SBNSoftware/sbnobj/pull/115/commits/a97b2773d58f57dcb8dbc311613c4a227ed1c361